src/Server/Storage/AzureTable.fs

```diff
@@ -72,29 +74,32 @@ let saveWishListToDB connectionString wishList = async {
         operation
     
     let! booksTable = getBooksTable connectionString
-    return! booksTable.ExecuteBatchAsync batch |> Async.AwaitTask |> Async.Ignore }
+    let! _ = booksTable.ExecuteBatchAsync batch 
```

Comment:
Were you not able to return the unit!? was type inference broken or compile/runtime error?

Reply:
yes Task.Ignore is missing